docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -813,4 +813,78 @@ Returns/Downloads a `tar` file which contains a `.meta` and a `.log` text
 files, and a `.tar.gz` of the raw DICOM data as acquired during the candidate
 scanning session, and as retrieved from `/candidates/$CandID/$Visit/dicoms`.
 
-Only `GET` is currently supported.
+```
+PUT /candidates/$CandID/$VisitLabel/dicoms/$Filename
+X-Is-Phantom: bool
+```
+
+Upload a tarred set of DICOM files that is compressed as a `.zip`,`.tar.gz` or 
+`.tgz`. $Filename must be the full name of the fileset and its extension.
+
+$Filename must be named according to the format 
+$PSCID_$CandID_$VisitLabel.ext where ext is one of the extensions above.
+
+X-Is-Phantom is a required header that must be sent as part of the request. 
+
+X-Is-Phantom must be set to `1` if the fileset is a phantom.
+X-Is-Phantom must be set to `0` for a fileset that is not a phantom.
+
+Returns JSON data upon success having the form:
+
+```js
+{
+    "status":"uploaded",
+    "mri_upload_id":"3"
+}
+```
+
+mri_upload_id is the Upload ID for the DICOM fileset.
+
+A file that has been succesfully uploaded will live in the location that is 
+specified in Loris for 'MRI-Upload Directory' in the "Paths" configuration.
+
+### 5.3 Process DICOM fileset
```

Comment:
I would add the following here.

```
GET /candidates/$CandID/$VisitLabel/dicoms/$Filename/processes
```
return the processes for that  DICOM fileset.

The response body format will be
```js
{'Processes': [
    {
    "process_type":"mri_upload", 
    "process_id": 123,
    "status": "started|running|complete|failed",
    },
    {
    "process_type":"mri_upload",
    "process_id": 124,
    "status": "started|running|complete|failed",
    }
]}
```